src/components/interaction-designer/clipboard/block/SelectOneResponseBlock.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="card" :class="{'disabled-block': !isFocused}">
+    <div class="card-body sm-padding-below font-roboto">
+      <h4 class="card-title font-weight-regular pl-0 text-color-title">{{prompt.block.label}}</h4>
+      <p class="card-text">
+        {{getContent}}
+        <i class="bi bi-pencil-fill"></i>
+      </p>
+
+    <div class="form-group">
+      <div v-for="(option, index) in options" :key="index" class="form-check">
+        <input
+          type="radio"
+          name="select-one"
+          class="form-check-input"
+          :class="{'is-invalid': errorMsg}"
+          :value="option.key"
+          :disabled="!isFocused"
+          :id="index"
+          v-model="selectedItem"
+          @change="checkIsValid"
+        />
+        <label class="form-check-label" :for="index">{{option.value}}</label>
+      </div>
+      <div v-if="errorMsg" style="color: #dc3545">
```

Comment:
Done!  The problem with using `.invalid-feedback` was that it was not under the same parent div, hence it would not work as expected. I have used `.text-danger` instead.